projects/slap_manipulation/scripts/add_detic_features_to_h5.py

```diff
@@ -0,0 +1,122 @@
+import glob
```

Comment:
we should make sure the right headers are on our files - can copy + paste license info from something else, like stretch.py